src/Sentry/Internal/Extensions/CollectionsExtensions.cs

```diff
@@ -29,9 +29,11 @@ internal static class CollectionsExtensions
         }
     }
 
+#if !NET8_0_OR_GREATER
```

Comment:
Technically not needed yet, will be needed when adding .NET 8: https://github.com/getsentry/sentry-dotnet/pull/2617/files#diff-ebf7bc6beaaf9dacfb725d68b7126baccbe823abd28dda757213831c04639774